laravel/app/Http/Controllers/StockController.php

```diff
@@ -3,6 +3,9 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use App\order;
```

Comment:
クラス名は大文字が良いので下記に修正をお願いします
```suggestion
use App\Order;
```